gooey/python_bindings/gooey_decorator.py

```diff
@@ -33,7 +33,10 @@ def Gooey(f=None,
           optional_cols=2,
           dump_build_config=False,
           load_build_config=None,
-          monospace_display=False):
+          monospace_display=False,
+          progress_regex=None,
+          progress_expr=None,
+          progress_animation=True):

```

Comment:
Leave it where it is. I like where you put it. It should just have the option of being hidden. 

There are certainly a subset of programs that shouldn't be casually stopped mid-execution by a user. My personal use-case for Gooey has always been for targeting non-techies. I don't want them to have `taskill` at their fingertips anytime they get impatient : )